core/src/test/scala/tech/beshu/ror/unit/boot/BuildInfoReaderTest.scala

```diff
@@ -0,0 +1,22 @@
+package tech.beshu.ror.unit.boot
+
+import monix.execution.Scheduler.Implicits.global
+import org.scalatest.WordSpec
+import tech.beshu.ror.es.BuildInfoReader
+
+class BuildInfoReaderTest extends WordSpec {
+
+  import org.scalatest.Matchers._
+
+  "BuildInfoReader" should {
+    "fail create from nonexistent file" in {
+      val error = BuildInfoReader.create("/dontexist.properties").failed.get
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Usage of get on optional type.](https://app.codacy.com/manual/scarduzio/elasticsearch-readonlyrest-plugin/pullRequest?prid=4718967)